src/components/Navigation/Navigation.jsx

```diff
@@ -119,9 +119,19 @@ export default class Navigation extends React.Component {
       });
 
       window.addEventListener('keyup', e => {
-        if (e.which === 9 && e.target.classList.contains('navigation__search-input')) {
-          this._openSearch();
+
+        switch(e.which) {
+          case 9: // `tab` key
+            if (e.target.classList.contains('navigation__search-input')) this._openSearch();
+            break;
+          case 191: // `/` key
+            this._openSearch(true);
```

Comment:
Suggest we introduce a var to track search bar's state, and when its open dont call `this._openSearch(true);` when user presses `/` for example when typing `/` into search input 
```
!this.searchIsOpen  && this._openSearch(true);
```

Reply:
Thanks for that, I have made some to reflect this.